config/locales/en.yml

```
@@ -1,36 +1,63 @@
-# Files in the config/locales directory are used for internationalization
-# and are automatically loaded by Rails. If you want to use locales other
-# than English, add the necessary files in this directory.
-#
-# To use the locales, use `I18n.t`:
-#
-#     I18n.t 'hello'
-#
-# In views, this is aliased to just `t`:
-#
-#     <%= t('hello') %>
-#
-# To use a different locale, set it with `I18n.locale`:
-#
-#     I18n.locale = :es
-#
-# This would use the information in config/locales/es.yml.
-#
-# The following keys must be escaped otherwise they will not be retrieved by
-# the default I18n backend:
-#
-# true, false, on, off, yes, no
-#
-# Instead, surround them with single quotes.
-#
-# en:
-#   'true': 'foo'
-#
-# To learn more, please read the Rails Internationalization guide
-# available at https://guides.rubyonrails.org/i18n.html.
-
 en:
   start_page:
     title: Offer support as a business to help with coronavirus
     body: Use this service to share the types of things your business can help with or might be able to help with. You'll be contacted with more information about how you can help.
     button_label: Start now
+  coronavirus_form:
+    errors:
+      heading: "There is a problem"
+      radio_field: "Select %{field}"
+      checkbox_field: "Select at least one %{field}"
+      missing_mandatory_text_field: "Enter %{field}"
+      invalid_money: "Total amount awarded must be an amount of money, like 15000"
```

It might be worth making this one a little more generic:
```suggestion
      invalid_money: "%{field} must be an amount of money, like 15000"
```